api/SourceBuffer.json

```diff
@@ -446,10 +446,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/SourceBuffer/audioTracks",
           "support": {
             "chrome": {
-              "version_added": "23"
+              "version_added": "49"
```

Comment:
Testing in https://software.hixie.ch/utilities/js/live-dom-viewer/?saved=8419, audioTracks and videoTracks aren't shipped in Chrome yet. It's all still behind a flag: https://chromium.googlesource.com/chromium/src/+/9670bbbca886d7b15084ecec18601a23c42066b9/third_party/blink/renderer/platform/runtime_enabled_features.json5#181

I've reopened https://github.com/mdn/browser-compat-data/issues/5781.